### PR TITLE
mailmap update

### PR DESCRIPTION
**Pull request by Helianthella**
if you know the github username of any contributor from the `missing from github` section ([L45](https://github.com/mekolat/Hercules/blob/e8766a67a2b15b602e41fc807cdf99a4a41e9a03/.mailmap#L45)), please tell so I can add them

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1857)

File: .mailmap [hunk 1]
```diff
@@ -7,3 +7,96 @@
...
+# - mc_cameri
+# - evera
+# - SinSloth
+# - epoque11
+# - momacabu
+# - DracoRPG
+# - sketchyphoenix
+# - littlewolf
+# - eaac
+# - daegaladh
```
Comment (Lemongrass3110) [inline .mailmap]: @Daegaladh

File: .mailmap [hunk 2]
```diff
@@ -7,3 +7,96 @@
...
+# - ajarn
+# - greenboxal2
```
Comment (Lemongrass3110) [inline .mailmap]: @greenboxal

File: .mailmap [hunk 3]
```diff
@@ -7,3 +7,96 @@
...
+# - Vicious
+# - samuray22
+# - Kevin
+# - eathenabot
+# - shadow
+# - mc_cameri
+# - evera
+# - SinSloth
+# - epoque11
+# - momacabu
```
Comment (Lemongrass3110) [inline .mailmap]: This sounds like @macabu, but i could be wrong

File: .mailmap [hunk 4]
```diff
@@ -7,3 +7,96 @@
...
+
+
+
+# missing from github:
+# - skotlex
+# - L0ne_W0lf
+# - Lupus
+# - celest
+# - Lance
+# - zephyrus
```
Comment (dastgir) [inline .mailmap]: Afaik, Not on github

File: .mailmap [hunk 5]
```diff
@@ -7,3 +7,96 @@
...
+# - amber
+# - Inkfish
+# - toms
+# - shadowlady
+# - codemaster
+# - rud0lp20
```
Comment (dastgir) [inline .mailmap]: @malufett

File: .mailmap (hunk 3 above)
Comment (macabu) [reply .mailmap]: Hello! Yes, that's me (momacabu) :smile:

**Pull request merged**